pkgs/development/python-modules/customtkinter/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, setuptools
+, wheel
+, tkinter
+, darkdetect
+}:
+
+buildPythonPackage rec {
+  pname = "customtkinter";
+  version = "5.2.2";
+  format = "pyproject";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-/Y2zuvqWHJgu5gMNuoC0wuJYWGMHVrUTmG2xkRPY0gc=";
+  };
+
+  nativeBuildInputs = [ setuptools wheel ];
+  buildInputs = [ tkinter ];
+  propagatedBuildInputs = [ darkdetect ];
+
+  doCheck = false;
```

Comment:
Please add an explanation in the comment above.

Reply:
Done

Reply:
that's just not true https://github.com/TomSchimansky/CustomTkinter/tree/master/test

They are likely just not uploaded to pypi and we need to fetch from github to get them

Reply:
Okay, to clarify, I meant in the context of the `fetchPypi` as there are no tests that come from the PyPI.

If necessary, I could try packaging it from the source and include the tests?